standard/test/array_test.lua

```diff
@@ -141,4 +142,26 @@ function suite:testReduce()
 	self:assertDeepEquals(1, Array.reduce({2, 3, 5}, pow, 1))
 end
 
+function suite:testExtractValues()
```

Comment:
Add testcase to verify the functionality of default implementation

Reply:
added some using table.sort before the compare